packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -185,6 +277,34 @@ export class ProductCard extends PureComponent {
             break;
         }
 
+        if (type_id === BUNDLE) {
+            if (price_view === 'PRICE_RANGE' && dynamic_price) {
```

Comment:
Please move the constants to the config.
Or reuse them. Do the search by the code first.